tests/filecheck/parser-printer/builtin_types.xdsl

```diff
@@ -2,14 +2,15 @@
 
 
 builtin.module() {
-  func.func() ["sym_name" = "tuple_to_tuple", "function_type" = !fun<[!tuple<[!i32]>], [!tuple<[!i32]>]>, "sym_visibility" = "private"] {
-    ^0(%0 : !tuple<[!i32]>):
-      func.return(%0 : !tuple<[!i32]>)
-  }
-}
+  func.func() ["sym_name" = "tuple_to_tuple", "value" = !fun<[!tuple<[!i32]>], [!tuple<[!i32]>]>, "function_type" = !fun<[], []>] {}
+  // CHECK: "value" = !fun<[!tuple<[!i32]>], [!tuple<[!i32]>]>
```

Comment:
I don't quite get this change. AFAIU, you are using a function operation here to show that `!fun<[!tuple<[!i32]>], [!tuple<[!i32]>]>` can be parsed/printed. Why use a function operation? Aren't there easier operations to do that with? You can give `sym_name` and `value` to any operation to keep the name of the test around as well, right?

I feel like I am confused by these tests each time I see them. Would it make sense to have an empty `test.testing` operation just to make these tests easier to understand?

Reply:
Yeah, I'll add a `test.testing` in a following PR, that should make things better!